### PR TITLE
Store imported attributes in a nested record.

### LINKED ISSUE #8: Try out using nested records for imported metadata
A `ScannedResource` could have a `imported_metadata` property which is another `Valkyrie::Resource`. When you re-import metadata you can just wipe out that property.

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #8

### CODE REVIEW

File: spec/models/scanned_resource_spec.rb [hunk 1]
```diff
@@ -11,9 +11,6 @@
       k.include PlumSchema
```
Comment (escowles) [inline spec/models/scanned_resource_spec.rb]: Should this go too?

**Comment by escowles** [comment]
The one thing I thought of is that there might be a big overlap between the fields that are moving to the nested resource here, and the EphemeraFolder.  But I think we can probably refactor PlumSchema to make it easy to include those in a reasonable way later.

**Comment by tpendragon** [comment]
@escowles Yeah, I figured we'd tackle that when we do MVW.

File: spec/models/scanned_resource_spec.rb (hunk 1 above)
Comment (tpendragon) [inline spec/models/scanned_resource_spec.rb]: Yup, removed.

**Pull request merged**